service-discovery/README.adoc

```diff
@@ -0,0 +1,324 @@
+[id="chap-using-servicediscovery"]
+=  Connecting {product-long} Service to your OpenShift cluster
+ifdef::context[:parent-context: {context}]
+:context: using-servicediscovery
+
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-long: Application Services
+:product: App Services
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://cloud.redhat.com/beta/application-services/streams/
+:property-file-name: app-services.properties
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As a developer of applications and services, you can connect Quarkus applications to Kafka instances in {product-long}. https://quarkus.io/[Quarkus^] is a Kubernetes-native Java framework made for Java virtual machines (JVMs) and native compilation, and optimized for serverless, cloud, and Kubernetes environments. Quarkus is designed to work with popular Java standards, frameworks, and libraries like Eclipse MicroProfile and Spring, as well as Apache Kafka, RESTEasy (JAX-RS), Hibernate ORM (JPA), Infinispan, Camel, and many more.
+
+OpenShift is an Enteprise Kuberentes Platform. When deploying Quarkus and any other type of Applications developers can use 
+https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator/[ServiceBinding] capability that would let 
+their applications to discover configuration automatically inside their OpenShift or Kubernetes instance.
```

Comment:
```suggestion
enable your applications to discover configuration automatically in your OpenShift or Kubernetes instance.
```

Reply:
Also, I'm not sure what "configuration" refers to here. Would it be correct to simply say that you use the Service Binding Operator to connect your application in OpenShift to a Kafka instance in Application Services?